backend/hitas/models/apartment.py

```diff
@@ -129,6 +133,30 @@ def is_new(self) -> bool:
 
         return no_first_purchase_or_in_the_future and no_first_sale_or_in_the_future
 
+    def sell_by_date(self):
```

Comment:
This should be changed to be a `property` or name be should changed to `get_sell_by_date` to signify that this is actually a method

Reply:
Made `sell_by_date` and `has_grace_period` into properties.